.circleci/config.yml

```diff
@@ -67,6 +81,13 @@ workflows:
               ignore:
                 - master
                 - /pull\/[0-9]+/
+      - publish-tag:
+          # Only run this job on git tag pushes
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /v[0-9]+(\.[0-9]+)*(-.*)*/
```

Comment:
@argvk what do you think about the versioning choice here? maybe we could build... all tags instead?

Reply:
The community is used to using "SemVer" : this makes it fairly easy to know if one is a new version or not compared to another ...

Reply:
@PascalBourdier yes, though for example other projects also maintain other tags, see https://hub.docker.com/_/alpine latest vs edge. Surely we can start with this :)

Reply:
ok I will change it

Reply:
yeah, this looks good. Perfectly fine to build only semver tags.